### PR TITLE
Ship Model Header File Added

### PR DESCRIPTION
**Pull request by asaxena5**
### Summary 

Added getters/setters for shipmodel.h

### Testing 

Builds

### Review Focus 

See if there's any other things ship model should have getters/setters for

### CODE REVIEW

File: build-apple/Sweetspace.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -127,6 +127,7 @@
 /* End PBXContainerItemProxy section */
```
Comment (asaxena5) [inline build-apple/Sweetspace.xcodeproj/project.pbxproj]: Didn't know if this file needs to be added so I included it, but I can be deleted if not needed.

File: source/ShipModel.h [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	virtual bool init();
+
+#pragma mark -
+#pragma mark Accessors
+	/**
+	 * Returns the list of donuts.
+	 *
+	 * @return the list of donuts.
+	 */
+	std::vector<std::shared_ptr<DonutModel>> getDonutList() { return donuts; }
```
Comment (mt-xing) [inline source/ShipModel.h]: Does returning the vector itself instead of a reference create a copy of the vector?

Comment (asaxena5) [reply source/ShipModel.h]: I don't think so, according to this: https://stackoverflow.com/questions/22655059/why-is-it-ok-to-return-a-vector-from-a-function

Comment (mt-xing) [reply source/ShipModel.h]: Maybe I'm reading it wrong, but that SO link seems to also say that the vector is being copied.

Comment (asaxena5) [reply source/ShipModel.h]: I'm not sure. I can change it to return a reference instead.

**Pull request merged**